NewsBook/Application/Response/NewsUpdateRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NewsBook.Mediator.Response
+{
+    public class NewsUpdateRequest
```

Comment:
Can we write the command class instead of this class?

And why are we using the Response class to parse the query/body from the API request?